docs/product-features/staking/how-to.md

```diff
@@ -1,3 +1,80 @@
 # How to stake on ALEX
 
-Comming soon.
\ No newline at end of file
+Staking on ALEX takes a few easy steps. In short, it consists of locking up your tokens temporarily to help secure the ALEX network and gain rewards in return. In staking, time is measured in cycles, and at the end of each cycle, you will be able to harvest your rewards. 
```

Comment:
Rewrite this. No secure alex network, etc..

Reply:
Discarded this section